landing page/js/main.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
You should enable `translate_tabs_to_spaces` here: https://www.sublimetext.com/docs/3/indentation.html